inc/manager.php

```diff
@@ -495,7 +504,7 @@ public function process_urls_from_content( $html ) {
 	 * @return array
 	 */
 	public function extract_image_urls_from_content( $content ) {
-		$regex = '/(?:http(?:s?):)(?:[\/\\\\|.|\w|\s|-])*(?:[' . Optml_Config::$chars . '])*\.(?:' . implode( '|', array_keys( Optml_Config::$extensions ) ) . ')(?:\?{1}[\w|=|&|\-|\.|:|;]*)?/';
+		$regex = '/(?:http(?:s?):)?(?:[\/\\\\|.|\w|-])*(?:[' . Optml_Config::$chars . '])*\.(?:' . implode( '|', array_keys( Optml_Config::$extensions ) ) . ')(?:[\?|%3F]{1}[\w|=|&|\-|\.|:|;|%]*)?/';
```

Comment:
Changed the group (http:|https:) to optional, added encoded form for `?` and allowed extra char `%` after extension

You can test the regex here:
https://regex101.com/r/Dd47Y1/3
and here:
https://regex101.com/r/Dd47Y1/4